### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by JeanmarcLaiAqovia**

### CODE REVIEW

**Review by timrog1 — APPROVED** [review]

**Review by timrog1 — APPROVED** [review]

**Review by timrog1 — APPROVED** [review]

**Review by timrog1 — APPROVED** [review]

File: Aqovia.PactProducerVerifier/Aqovia.PactProducerVerifier.nuspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Aqovia.PactProducerVerifier</id>
+    <version>1.0.3</version>
```
Comment (ma499) [inline Aqovia.PactProducerVerifier/Aqovia.PactProducerVerifier.nuspec]: Shouldn't this just specify the major and minor version with the patch version left at 0 and set by Appveyor?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Aqovia.PactProducerVerifier
+A utility for verifying producer code against all consumers on the Pact Broker
```
Comment (ma499) [inline README.md:2]: I would like to see more guidance on how to actually use this library, including simple examples or links to examples.

Comment (ma499) [inline README.md:2]: Can we add build status and nuget badges please?

**Review by ma499 — COMMENTED** [review]
Can we add some tests?

File: README.md (hunk 2 above)
Comment (JeanmarcLaiAqovia) [reply README.md:2]: Hi @ma499 - I've updated the readme.md file with some guidance and examples.

Comment (JeanmarcLaiAqovia) [reply README.md:2]: added the build status and nuget badge

**Pull request merged**